pavelib/assets.py

```diff
@@ -89,6 +90,36 @@ def applicable_sass_directories(systems=None):
     return applicable_directories
 
 
+class DebounceEventQueue(EventQueue):
+    """
+    Event queue that discards puts if they happen too quickly.
+    """
+    wait_seconds = 1
+
+    def __init__(self, *args, **kwargs):
+        EventQueue.__init__(self, *args, **kwargs)
+        self._last_updated = None
+
+    def _put(self, *args, **kwargs):
+        """
+        Only puts the item if the last put was at least `wait_seconds` ago.
+        """
+        now = datetime.now()
+        if not self._last_updated or (now - self._last_updated).total_seconds() > self.wait_seconds:
+            self._last_updated = now
+            EventQueue._put(self, *args, **kwargs)
+
+
+class DebouncePollingObserver(PollingObserver):
```

Comment:
Given that polling is the only thing that will work in in devstack (neither vboxfs or nfs support inotify), is it worth supporting anything else at this point? Does anyone develop edx-platform outside of a devstack?